gpcontrib/arenadata_toolkit/.gitignore

```diff
@@ -2,4 +2,4 @@
 /results/
 regression.diffs
 regression.out
-arenadata_toolkit--1.*.sql
+arenadata_toolkit--?.?.sql
```

Comment:
(rare scenario) In case we'll occasionally decide to modify "arenadata_toolkit--1.0.sql", the modifications will be ignored.
(more real scenario) In case we'll create 2.0 version, which will be incompatible with 1.x versions, the new script file will be also ignored.

Just remove extra line from git ignore and don't include auto-generated files to repo.